curriculum/challenges/english/02-javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console.english.md

```diff
@@ -10,7 +10,7 @@ forumTopicId: 301193
 You may have noticed that some freeCodeCamp JavaScript challenges include their own console. This console behaves a little differently than the browser console you used in the last challenge.
 The following challenge is meant to highlight the main difference between the freeCodeCamp console and your browser console.
 When you run ordinary JavaScript, the browser's console will display your <code>console.log()</code> statements the exact number of times it is called.
-The freeCodeCamp console will print your <code>console.log()</code> statements a short time after the editor detects a change in the script, as well as during every test.
+The freeCodeCamp console will print your <code>console.log()</code> statements a short time after the editor detects a change in the script, as well as during testing.
 The freeCodeCamp console is cleared before every test. So, you may miss a log, if the tests finish too quickly.
 If you would like to see every log for every test, run the tests, and open the browser console. If you would like to use the browser console, and want it to operate the same as the freeCodeCamp console, place <code>console.clear()</code> before any other <code>console</code> calls, to clear the browser console.
```

Comment:
```suggestion
If you would like to see every log for every test, run the tests, and open the browser console. If you would like to use the browser console, and want it to operate the same as the freeCodeCamp console, place <code>console.clear()</code> before any other <code>console</code> calls, to clear the browser console.

**Note:** `console.log`s inside functions are printed to the freeCodeCamp console whenever those functions are called, this can help debugging functions that are called during testing.
```
The note referred to above.

Reply:
Me nitpicking here.

Shouldn't it be `console.log`'s?

Reply:
Apostrophy 's'?  Nope, it's just plural logs, not possessive.

Reply:
Unfortunately, this targeted the old commit. So, I have added a new commit with all the current suggestions.